cfme/tests/cli/test_appliance_console.py

```diff
@@ -1065,8 +1065,38 @@ def test_appliance_console_vmdb_httpd():
         caseimportance: low
         caseposneg: negative
         initialEstimate: 1/12h
+        testSteps:
+            1. Stop the EVM service
+            2. press "ap"
+            3. press any key to continue
+            4. Press "7" for "Configure Database"
+            5. Press "4" for "Reset Configured Database"
+            6. Press "Y" for confirmation
+            7. Enter Region Number
+            8. Start EVM service
+        expectedResults:
+            1.
+            2.
+            3.
+            4.
+            5.
+            6.
+            7. Check "Database reset successfully" message
+            8. UI should be up and running
+
     """
-    pass
+    appliance = temp_appliance_preconfig_funcscope
+    command_set = ("ap", RETURN, "16", TimedCommand("Y", 120))
+    appliance.appliance_console.run_commands(command_set, timeout=30)
+
+    command_set = ("ap", RETURN, "7", "4", "Y", TimedCommand("99", 300))
+    result = appliance.appliance_console.run_commands(command_set, timeout=30)
+    assert "Database reset successfully" in result[-1], "DB reset failed"
+
+    command_set = ("ap", RETURN, "17", TimedCommand("Y", 120))
+    appliance.appliance_console.run_commands(command_set, timeout=30)
+
+    wait_for(lambda: appliance.wait_for_web_ui(), delay=10, timeout=300)
```

Comment:
No need to wrap this in a wait_for (or to use lambda if wait_for was necessary).  

The function is already waiting, and will raise exception.
https://github.com/ManageIQ/integration_tests/blob/master/cfme/utils/appliance/__init__.py#L1490

Reply:
Done